nix/pkgs/haskell/haskell.nix

```diff
@@ -148,36 +148,26 @@ let
         packages.plutus-core.components.benchmarks.update-cost-model.buildable = lib.mkForce false;
       })
       ({ pkgs, ... }: lib.mkIf pkgs.stdenv.hostPlatform.isWindows {
-        # Things that need plutus-tx-plugins
-	    packages.marlowe.components.library.buildable = lib.mkForce false;
-	    packages.marlowe-actus.components.library.buildable = lib.mkForce false;
-	    packages.marlowe-actus.components.tests.marlowe-actus-test.buildable = lib.mkForce false;
-	    packages.marlowe-dashboard-server.components.library.buildable = lib.mkForce false;
-	    packages.marlowe-playground-server.components.library.buildable = lib.mkForce false;
-	    packages.marlowe-symbolic.components.library.buildable = lib.mkForce false;
-	    packages.playground-common.components.library.buildable = lib.mkForce false;
-	    packages.playground-common.components.tests.playground-common.buildable = lib.mkForce false;
-	    packages.plutus-benchmark.components.library.buildable = lib.mkForce false;
-	    packages.plutus-benchmark.components.tests.plutus-benchmark-nofib-tests.buildable = lib.mkForce false;
-	    packages.plutus-benchmark.components.benchmarks.nofib.buildable = lib.mkForce false;
-	    packages.plutus-chain-index.components.library.buildable = lib.mkForce false;
-	    packages.plutus-chain-index.components.tests.plutus-chain-index-test.buildable = lib.mkForce false;
-	    packages.plutus-contract.components.library.buildable = lib.mkForce false;
-	    packages.plutus-contract.components.tests.plutus-contract-test.buildable = lib.mkForce false;
-	    packages.plutus-errors.components.library.buildable = lib.mkForce false;
-	    packages.plutus-ledger.components.library.buildable = lib.mkForce false;
-	    packages.plutus-ledger.components.tests.plutus-ledger-test.buildable = lib.mkForce false;
-	    packages.plutus-pab.components.library.buildable = lib.mkForce false;
-	    packages.plutus-pab.components.tests.plutus-pab-test-full.buildable = lib.mkForce false;
-	    packages.plutus-pab.components.tests.pab-test-light.buildable = lib.mkForce false;
-	    packages.plutus-playground-server.components.library.buildable = lib.mkForce false;
-	    packages.plutus-playground-server.components.tests.plutus-playground-server-test.buildable = lib.mkForce false;
-	    packages.plutus-use-cases.components.library.buildable = lib.mkForce false;
-	    packages.plutus-use-cases.components.tests.plutus-use-cases-test.buildable = lib.mkForce false;
-	    packages.web-ghc.components.library.buildable = lib.mkForce false;
-	    packages.web-ghc.components.tests.web-ghc-test.buildable = lib.mkForce false;
-	    # Needs agda
-	    packages.plutus-metatheory.components.library.buildable = lib.mkForce false;
+        packages = {
+          # Things that need plutus-tx-plugins
+          marlowe.package.buildable = false;
```

Comment:
Now *here's* an example of where a cabal `buildable` (or `isCross`?) flag would make life easier, since we already have the ability to set flags on a per-package basis!

Reply:
We should revisit this, but for now I think excluding them in `nix` makes the most sense.  It is kind of `haskell.nix` that is at fault to provide a cross compiler with support for plugins.

If we did do it with a cabal flag I wonder if `no-plutus-tx-plugin` a nicer flag to have.  It might be a pain to maintain that in all the `.cabal` files though.